src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorReferenceProvider.ts

```diff
@@ -29,41 +28,25 @@ export class RazorReferenceProvider
             projection.uri,
             projection.position) as vscode.Location[];
 
-        const results = new Array<vscode.Location>();
```

Comment:
I imagine all of these deletions can also occur for the `RazorDefinitionProvider` and `RazorImplementationProvider` correct? If so, don't need to block this PR, it's just work we'll have to do.